pros/serial/ports/v5_wireless_port.py

```diff
@@ -0,0 +1,36 @@
+from typing import *
+
+from pros.serial.devices.vex.v5_device import V5Device
+from pros.serial.ports import BasePort, DirectPort
+
+
+class V5WirelessPort(BasePort):
+    def __init__(self, port):
+        self.buffer: bytearray = bytearray()
+
+        self.port_instance = DirectPort(port)
+        self.device = V5Device(self.port_instance)
+        # TODO: add version check for this feature? or should that be done by the terminal?

```

Comment:
nit: remove this todo if we're not going to do it. I don't think this is the type of thing that we'll add later.

Reply:
this made more sense in 2019 when the feature had just come out 😓